src/plugins/inspectorViews/annotations/AnnotationsInspectorView.vue

```diff
@@ -191,8 +192,14 @@ export default {
     },
     async loadAnnotationForTargetObject(target) {
       const targetID = this.openmct.objects.makeKeyString(target.identifier);
+      if (this.abortController !== null) {
```

Comment:
We should add a comment here that (I think) we're aborting if the user changed target options.